spyder/plugins/updatemanager/workers.py

```diff
@@ -12,6 +12,7 @@
 import os.path as osp
 import platform
 import shutil
+from subprocess import run
```

Comment:
```suggestion
import subprocess
```

`run` is such a common word that it could be easily masked by another variable. So, I think it's better to use this function by calling `subprocess.run` instead.